lib/offline_database/offline_db_handler.dart

```diff
@@ -489,9 +488,9 @@ class OfflineDbHandler {
       /* WeekDayColors is a list in SettingsModel,
          which means that they have to be saved in its own table */
       if (settings.weekDayColors != null) {
-        for (WeekdayColorModel weekdayColor in settings.weekDayColors) {
-          insertSettingsWeekDayColor(settingsId, weekdayColor);
-        }
+        //for (WeekdayColorModel weekdayColor in settings.weekDayColors) {
```

Comment:
hvorfor udkommenteres det, og hvad med if'en.
skal der ikke bare stå:
```
if (settings.weekDayColors == null)
   _updateUserSettings(userId, settings);
```

Reply:
mere fordi der er nogle der arbejder med at opdatere offline databasen så istedet for at slette det så de ikke kan se hvad der har stået der så lader jeg det blive og det er noget de har spurgt mig om at lade blive